app/navigation/AuthorisedOfficialsNavigator.scala

```diff
@@ -71,13 +71,15 @@ class AuthorisedOfficialsNavigator @Inject()(implicit frontendAppConfig: Fronten
       case Some(true) => routes.DeadEndController.onPageLoad() // TODO redirect to next page once created
       case Some(false) => index match {
         case 0 => authOfficialRoutes.AddedOneAuthorisedOfficialController.onPageLoad()
-        case 1 => authOfficialRoutes.AddedSecondAuthorisedOfficialController.onPageLoad()}
-      case _ =>  routes.SessionExpiredController.onPageLoad()
+        case 1 => authOfficialRoutes.AddedSecondAuthorisedOfficialController.onPageLoad()
+        case _ => routes.SessionExpiredController.onPageLoad() // TODO redirect to next page once created
```

Comment:
what's the next page you're addressing here? isn't a case where it's not 0 or 1 impossible in this situation (since one can only add 2 authorised officials)?

Reply:
this is for url tamporing, if someone add any other index